site.js

```diff
@@ -1,16 +1,32 @@
+/**
+* Dependencies:
+*    Mapbox JS SDK 2.4.0
```

Comment:
Change to `mapbox.js` - we use the term `SDK` for our mobile libraries (Android, iOS), not for our JavaScript libraries (mapbox.js and Mapbox GL JS)